src/App.tsx

```diff
@@ -31,6 +40,18 @@ export default function App() {
     CoverageType[]
   >([]);
   const [AllPokemon, setAllPokemon] = React.useState<Pokemon[]>([]);
+
+  // Navigate to pokedex page on pressing cmd/ctrl-f
+  useShortcut(
+    (event: KeyboardEvent): void => {
```

Comment:
Can be simplified to `event => {` because the type signature is inferred based on the definition of `useShortcut`